test/Vagrantfile

```diff
@@ -91,6 +91,7 @@ Vagrant.configure("2") do |config|
                 sh.args = [
                     "k8s#{i}", "192.168.36.1#{i}", "#{$K8S_VERSION}",
                     "#{$IPv6}", "#{$CONTAINER_RUNTIME}"]
+                sh.env = { "CNI_INTEGRATION" => ENV['CNI_INTEGRATION'] }
```

Comment:
Why you need this env? I do not think that it's really needed at all.

Reply:
Because it's used in `test/provision/k8s_install.sh`

Reply:
To be consistent then, could you use as a parameter like the other ones?

```
NODE=$1
IP=$2
K8S_VERSION=$3
IPv6=$4
CONTAINER_RUNTIME=$5
```